_documentation/general/backup.md

```diff
@@ -17,4 +17,4 @@ Sync your subscriptions via [gpodder.net](https://gpodder.net/). This will save
 Export the AntennaPod database. This is experimental, and you should make sure you have the same version of AntennaPod on both the old and new phone. After importing all data is restored; you will only need to re-download episodes.
 
 ## OPML
-OPML files allow to transfer your subscriptions to another podcast app. You will loose any podcast- or episode-specific data (like if a specific episode has been played) and all app settings. To export your file, go to `Settings` » `Storage` »  `Import/Export`. Be sure to click `Send…` to save the file in another folder on your phone or somewhere else, because the original file will be deleted when you uninstall AntennaPod.
+OPML files allow to transfer your subscriptions to another podcast app. You will loose any podcast- or episode-specific data (like if a specific episode has been played) and all app settings. To export your file, go to `Settings` » `Storage` »  `Import/Export`. Be sure to click `Send…` to save the file in another folder on your phone or somewhere else, because the original file will be deleted when you uninstall AntennaPod. When you import the file on another device, it will display a list with checkboxes and the podcasts you select will be added to your existing subscriptions. 
```

Comment:
> it will display a list with checkboxes

Not really happy about that part, to be honest. Could you please also write that you can select which podcasts to import? Also, "it" sounds like it could mean any podcast app but we only know that about AntennaPod.